sympy/core/add.py

```diff
@@ -68,7 +68,7 @@ def _unevaluated_Add(*args):
 
 class Add(Expr, AssocOp):
 
-    __slots__ = []
+    __slots__ = ['_constructor_postprocessor_dict']
```

Comment:
Because you should try to avoid adding implementation specific (list, dict, ...) pre- or postfixes to variable names and instead use general names (plural for lists//sequences and 'mapping' for dict-like objects).